server/run-dynamodb-local.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+DYNAMODB_CONTAINER_NAME=$1
+ENDPOINT_URL=$2
```

Comment:
Would be good to document what to pass in for these arguments and maybe error if they're not set?

Reply:
I think we can instead hard code those values, wdyt? Not sure we really need that flexibility